addons/test_mail/tests/test_performance.py

```diff
@@ -611,7 +611,7 @@ def test_complex_tracking_subscription_create(self):
         customer_id = self.customer.id
         user_id = self.user_portal.id
 
-        with self.assertQueryCount(__system__=105, emp=106):
+        with self.assertQueryCount(__system__=111, emp=112):
```

Comment:
We have 2 more query per call of `_split_by_mail_configuration`, 1 to search the record and an other to read the fields.
Here we have more queries because the `send` method of `mail.mail` is called 2 times (so we have 4 more queries).

So yes, maybe the ORM should retrieve the fields in the cache for the second of `_split_by_mail_configuration` call, but I do not really see what can be done in this task to improve that